README.md

```diff
@@ -1,4 +1,4 @@
-# Vault Plugin: Azure Secrets Backend [![Build Status](https://travis-ci.org/hashicorp/vault-plugin-secrets-azure.svg?branch=master)](https://travis-ci.org/hashicorp/vault-plugin-secrets-azure)
+# Vault Plugin: Azure Secrets Backend [![Build Status](https://travis-ci.org/hashicorp/vault-plugin-secrets-azure.svg?branch=main)](https://travis-ci.org/hashicorp/vault-plugin-secrets-azure)
```

Comment:
We probably could get rid of the travis badge since it's not built there anymore.